src/Directive/AdmonitionDirective.php

```diff
@@ -20,9 +20,7 @@ public function processSub(Parser $parser, ?Node $document, string $variable, st
         $wrapperDiv = $parser->renderTemplate(
             'directives/admonition.html.twig',
             [
-                // a bit strange, but on the old markup we literally
-                // had a class of 'admonition-"
-                'name' => '',
+                'name' => 'default',
```

Comment:
ah, cool! I forgot I did this on purpose! And now we can't do whatever we want. WEEEEEE

Reply:
welp, that was fun.. ha :)
fixed

Reply:
Hi @jrushlow 

I don't see this update anymore?

Reply:
I reverted the commit that added `default`.. did i misunderstand @weaverryan e.g. we must have
```
<div class="admonition-">
``` 
instead of
```
<div class="admonition-default">
```

Reply:
Ah, sorry, I confused things. I DO want `admonition-default`.

My comment was just me realizing that I had originally done `admonition-` on purpose to exactly match the markup already used on Symfony.com. But due to how we're using this lib, we no longer need to "blindly make the same markup as we currently have on Symfony.com". We're free to make markup that is what we want and correct. So yes, please DO make the change to do `admonition-default`.

Reply:
fixed